### ISSUE TITLE
TIP-318: Adapt to Ethereum London Release

### ISSUE THREAD

**Issue by Lredhdx** [tronprotocol/tips#318]
```
tip: 318
title: Adapt to Ethereum London Release
author: neo.hong@tron.network
status: Draft
type: Standards Track
category: VM
created: 2021-08-16
```

## Simple Summary

To Adapt to Ethereum London Release.

## Abstract

Ethereum London Release adds some features. To provide millions of global developers with a custom-built blockchain system. Tron is supposed to adapt to it.

## Implementation

To Adapt to Ethereum London Release. We need add `BASEFEE` opcode in tvm and reject new contracts starting with the `0xEF` byte.

**Comment by Kusmie00** [comment]
https://github.com/tronprotocol/tips/issues/318#issue-971325946

**Comment by yanghang8612** [comment]
Close this issue as it is implemented by [GreatVoyage-v4.4.0(Rousseau)](https://github.com/tronprotocol/java-tron/releases/tag/GreatVoyage-v4.4.0)
Check TIP detail at [TIP-318](https://github.com/tronprotocol/tips/blob/master/tip-318.md)
Check implementation PR at https://github.com/tronprotocol/java-tron/pull/4032
